frontend/src/wizards/Argo/ArgoWizard.tsx

```diff
@@ -303,7 +303,7 @@ export function ArgoWizard(props: ArgoWizardProps) {
           <Section label={t('General')}>
             <WizTextInput
               path="metadata.name"
-              label={t('ApplicationSet name')}
+              label={`ApplicationSet ${t('name')}`}
```

Comment:
I think just "Name" would be good here. @fxiang1 that OK?

If you want to include a non-translated word in a translation string, you should use i18n interpolation, eg. `"applicationSetName": "{{applicationSet}} name"` rather than a JavaScript template string. That way the translator can put `"name of {{applicationSet}}"` in a different language in case that's how the language works.

Reply:
Yep, that's good with me. It aligns with the other wizards.